### DISCUSSION TITLE
如何设置小说从主页面打开而不是侧边栏？

### DISCUSSION THREAD

**Discussion by BeiQingHong** [aooiuu/any-reader#49] [Q&A]
如何设置小说从主页面打开而不是侧边栏？

**Comment by aooiuu** [comment]
可以先点击**侧边栏**底部的**大搜索**图标，这会在右侧**编辑器**区域打开一个**Tab**，在新打开的页面里面操作就可以了

目前主页面和侧边栏的代码是一样的，功能也是一样的，**大多数都是在当前的页面跳转**。
特殊情况包括：
1.大搜索会跳转到主页面
2.发现页会跳转到主页面
3.规则编辑会跳转到主页面
接着在主页面内操作会继续在当前的主页面跳转

当然如果有合适的交互可以探讨如何去设置定义跳转方式

**Comment by aooiuu** [comment]
章节列表添加了从主页面打开的按钮
